File: electronics-sales/src/main/java/com/electronicssales/services/ImportInvoiceService.java
package com.electronicssales.services;

import com.electronicssales.entities.ImportInvoice;
import com.electronicssales.models.dtos.ImportInvoiceDto;

public interface ImportInvoiceService {

    ImportInvoice saveImportInvoice(ImportInvoiceDto importInvoice);
    
}

File: electronics-sales-frontend/src/app/services/import-invoice.service.ts
import { ImportInvoiceDto } from './../models/dtos/import-invoice.dto';
import { HttpClient } from '@angular/common/http';
import { Injectable } from '@angular/core';

@Injectable({
  providedIn: 'root'
})
export class ImportInvoiceService {

  static readonly BASE_REQUEST = '/api/import-invoices'

  constructor(private http: HttpClient) { }

  create(importInvoiceDto: ImportInvoiceDto) {
    return this.http.post<ImportInvoiceDto>(`${ImportInvoiceService.BASE_REQUEST}`, importInvoiceDto);
  }
}
